### ISSUE TITLE
proxy-mysql-default-version set to 5.6.x will result in failure to execute SQL

### ISSUE THREAD

**Issue by linghengqian** [apache/shardingsphere#21453]
## Feature Request

**For English only**, other languages will not accept.

Please pay attention on issues you submitted, because we maybe need more details. 
If no response anymore and we cannot make decision by current information, we will **close it**.

Please answer these questions before submitting your issue. Thanks!

### Is your feature request related to a problem?

- Yes，refer to https://github.com/apache/shardingsphere/issues/21442 .

### Describe the feature you would like.

- In the Docker Image of https://github.com/apache/shardingsphere/pkgs/container/shardingsphere-proxy/44714726?tag=f4080b1b3ddcaa9eb15e87e50a6fb0d2fe2baa7f, if I set `props.proxy-mysql-default-version` of `server.yaml` as follows.
```yaml
props:
    proxy-mysql-default-version: 5.6.51
```
- When executing a statement like `use mysql;` in Jetbrains DataGrip, the error `For input string: ""` will occur, the SQL execution fails, and the server log of ShardingSphere Proxy will not show any errors.
- Is this expected behavior considering the default value of this property is `5.7.22`? Or should the documentation state that the version number set by this property should be greater than or equal to `5.7.0`? Refer to https://shardingsphere.apache.org/document/5.2.0/en/user-manual/shardingsphere-proxy/yaml-config/props/ .
- ![image](https://user-images.githubusercontent.com/20187731/194856775-e0126904-362c-4013-84d7-a31742c0b0f3.png)

**Comment by TeslaCN** [comment]
Have you configured any data sources?

**Comment by linghengqian** [comment]
- I tested with the database configured in `config-*.yaml`.  It might be a little hard for me to provide such unit tests as Oracle doesn't seem to maintain a Docker Image for MySQL 5.6.

**Comment by linghengqian** [comment]
- I was intent on writing a unit test for this issue, but got blocked by https://github.com/apache/shardingsphere/issues/22737 at first.

**Comment by linghengqian** [comment]
- Close this issue because ShardingSphere Proxy 5.3.1 does not show the same Error Log as any previous version. I will reopen a new issue after investigating.
- This is most likely related to an update of Jetbrains DataGrip.
